src/screens/WalletScreen.js

```diff
@@ -195,20 +194,6 @@ export default function WalletScreen() {
     }
   }, [portfolios, portfoliosFetched, trackPortfolios, userAccounts.length]);
 
-  useEffect(() => {
```

Comment:
some peeps might not like this change but seems fine for now since this is a weird way to do it  🤷‍♂️

Reply:
"some peeps" aka christian :) yeah I have a follow up todo to look into sending this list up from the brief memo selectors